addons/irlight/CfgWeapons.hpp

```diff
@@ -0,0 +1,393 @@
+        scale[] = {0.25, 0.25, 1}; \
+        class Attenuation { \
+            constant = 1; \
+            linear = 0; \
+            quadratic = 0.008; \
+            start = 1; \
+            hardLimitStart = 220; \
+            hardLimitEnd = 250; \
+        }; \
+    }
+#define DBAL_A3_FLASHLIGHT_LR \
+    class Flashlight { \
+        color[] = {1, 1, 1}; \
+        ambient[] = {1, 1, 1}; \
+        size = 1; \
+        innerAngle = 10; \
+        outerAngle = 12; \
+        position = "laser pos"; \
+        direction = "laser dir"; \
+        useFlare = 1; \
+        flareSize = 1.4; \
+        flareMaxDistance = 200; \
+        coneFadeCoef = 6; \
+        intensity = 200; \
+        irLight = 1; \
+        volumeShape = "a3\data_f\VolumeLightFlashlight.p3d"; \
+        scale[] = {0.25, 0.25, 1}; \
+        class Attenuation { \
+            constant = 1; \
+            linear = 0; \
+            quadratic = 0.001; \
+            start = 1; \
+            hardLimitStart = 570; \
+            hardLimitEnd = 600; \
+        }; \
+    }
+#define SPIR_FLASHLIGHT(hardLimitStart,hardLimitEnd) \
+    color[] = {1, 1, 1}; \
+    ambient[] = {1, 1, 1}; \
+    size = 1; \
+    position = "flash dir"; \
+    direction = "flash"; \
+    useFlare = 1; \
+    flareSize = 1.4; \
+    flareMaxDistance = 100; \
+    irLight = 1; \
+    volumeShape = "a3\data_f\VolumeLightFlashlight.p3d"; \
+    class Attenuation { \
+        constant = 1; \
+        linear = 0; \
+        quadratic = 0.02; \
+        start = 1; \
+        hardLimitStart = hardLimitStart; \
+        hardLimitEnd = hardLimitEnd; \
+    }
+#define SPIR_FLASHLIGHT_LR(hardLimitStart,hardLimitEnd) \
+    color[] = {1, 1, 1}; \
+    ambient[] = {1, 1, 1}; \
+    size = 1; \
+    position = "flash dir"; \
+    direction = "flash"; \
+    useFlare = 1; \
+    flareSize = 1.4; \
+    flareMaxDistance = 100; \
+    irLight = 1; \
+    volumeShape = "a3\data_f\VolumeLightFlashlight.p3d"; \
+    class Attenuation { \
+        constant = 1; \
+        linear = 0; \
+        quadratic = 0.002; \
+        start = 1; \
+        hardLimitStart = hardLimitStart; \
+        hardLimitEnd = hardLimitEnd; \
+    }
+#define POINTER \
+    class Pointer { \
+        irLaserPos = "laser pos"; \
+        irLaserEnd = "laser dir"; \
+        irDistance = 5; \
+    }
+
+class CfgWeapons {
+    class acc_pointer_IR;
+    class acc_flashlight;
+    class InventoryFlashlightItem_Base_F;
+
+    // DBAL-A3 (red pointer)
+    class ACE_DBAL_A3_Red: acc_pointer_IR {
+        author = ECSTRING(common,ACETeam);
+        displayName = CSTRING(DBAL_A3_Red);
+        descriptionUse = CSTRING(DBAL_A3_DescriptionUse);
+        descriptionShort = CSTRING(DBAL_A3_DescriptionShort);
+        MRT_SwitchItemNextClass = "ACE_DBAL_A3_Red_IP";
+        MRT_SwitchItemPrevClass = "ACE_DBAL_A3_Red_VP";
+        MRT_SwitchItemHintText = CSTRING(Mode_IRDual);
+        GVAR(baseClass) = "ACE_DBAL_A3_Red";
```

Comment:
```suggestion
        GVAR(baseClass) = "ACE_DBAL_A3_Red";
        baseWeapon = "ACE_DBAL_A3_Red";
```

I know GVAR(baseClass) is there but I don't want to add another property to the Arsenal lookup.